app/assets/javascripts/profile/manage.js.coffee

```diff
@@ -33,4 +33,13 @@ $(document).on('turbolinks:load', ->
         data: $(this).sortable('serialize')
       )
   )
+
+  $('tbody#other_publications').sortable(
+    update: (event, ui) ->
+      $.ajax(
+        method: 'PUT',
+        url: '/authorships/sort',
+        data: $(this).sortable('serialize')
+      )
+  )
 )
```

Comment:
Since the order of the other-publications' presentation in the public profile is bound to the default order of the publication types then maybe it would be best to just remove the ability to order the other-publications for now.   They seem to be less important to the faculty anyways.  If someone requests the ability to order them, then we can work on it then.  I'll add a github issue to consider it.